src/test/java/com/karankumar/bookproject/backend/utils/PredefinedShelfUtilsTest.java

```diff
@@ -40,6 +55,10 @@
     private static Book book3 = new Book("someOtherTitle", NO_AUTHOR, readShelf);
     private static Book book4 = new Book("yetAnotherTitle", NO_AUTHOR, didNotFinishShelf);
 
+    private static final List<String> PREDEFINED_SHELVES = List.of("To read", "Reading", "Read", "Did not finish");
```

Comment:
Instead of hardcoding the names of the predefined shelves, could you please use the `getPredefinedShelfNamesAsStrings()` utility method in `PredefinedShelfUtils`?